### PR TITLE
Setup Percy

### PR DESCRIPTION
**Pull request by jdamiba**
The purpose of this PR is to setup CircleCI so that it takes Percy screenshots of this repo when working on feature branches.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    
+      - save_cache:
+          key: gem-cache-v1-{{ arch }}-{{ .Branch }}-{{ checksum "Gemfile.lock" }}
+          paths:
+            - vendor/bundle
+
+      - run:
+          name: deployment-help-repo
+          command: |
+            git config --global user.email "accounts@plot.ly"
```
Comment (nicolaskruchten) [inline .circleci/config.yml]: no need for git configs, we're not committing from ehre

**Comment by nicolaskruchten** [comment]
💃 once you remove those few unnecessary bits

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      ignore:
+        - gh-pages
+
+    docker:
+      # a packaged system that has the instructions for creating a running container.
+      - image: circleci/ruby:2.3.3
+
+    # actions that need to be taken to perform your job
+    steps:
+      - add_ssh_keys:
```
[the remark this answers is not in the document]
Comment (jdamiba) [reply .circleci/config.yml]: Removed.

File: .circleci/config.yml (hunk 1 above)
Comment (jdamiba) [reply .circleci/config.yml]: Removed.

**Pull request merged**